cmd/ingress-perf.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/cloud-bulldozer/ingress-perf/pkg/config"
 	_ "github.com/cloud-bulldozer/ingress-perf/pkg/log"
 	"github.com/cloud-bulldozer/ingress-perf/pkg/runner"
+	"github.com/cloud-bulldozer/kube-burner/pkg/version"
```

Comment:
can you create the version package rather tan importing the kube-burner one?

Reply:
Sure!

Reply:
>can you create the version package rather tan importing the kube-burner one?

Any reason why we do not want to do it this way?

Reply:
Updated:

```
./bin/ingress-perf version
Version: version-branch
Git Commit: e68d9ce3d769d30339fafc83a4d4aa9abcfbccb4
Build Date: 
Go Version: go1.20.1
OS/Arch: darwin amd64
```

Reply:
> > can you create the version package rather tan importing the kube-burner one?
> 
> Any reason why we do not want to do it this way?

I think we shouldn't use kube-burner as a library, it's a bit weird, don't you think?